src/components/visualizations/chart/chartOptions/bulletChartOptions.ts

```diff
@@ -35,39 +39,33 @@ const getSeriesItemData = (
     seriesItem: string[],
     measureGroup: Execution.IMeasureGroupHeader["measureGroupHeader"],
     seriesIndex: number,
-    bucketsLocalIdentifiers: IBucketLocalIdentifier[],
-): IPointData[] =>
+    measureBucketsLocalIdentifiers: Array<VisualizationObject.IBucket["localIdentifier"]>,
+) =>
     seriesItem.map((pointValue: string) => ({
-        ...getValue(pointValue, seriesIndex, bucketsLocalIdentifiers),
+        ...getValue(pointValue, seriesIndex, measureBucketsLocalIdentifiers),
         format: unwrap(measureGroup.items[seriesIndex]).format,
         marker: {
             enabled: pointValue !== null,
         },
         name: unwrap(measureGroup.items[seriesIndex]).name,
     }));
 
-const getPrimarySeriesMaxPointWidth = (comparativeMeasurePresent: boolean) => {
-    if (comparativeMeasurePresent) {
-        return (MAX_POINT_WIDTH / 4) * 3;
+const getPrimarySeriesMaxPointWidth = (onlyPrimaryMeasure: boolean) => {
+    if (!onlyPrimaryMeasure) {
+        return MAX_POINT_WIDTH * 0.75;
```

Comment:
It would be great to extract `0.75` to constant and give it a name which has meaning